tracer.go

```diff
@@ -140,6 +136,17 @@ func NewTracer(
 	return t, t
 }
 
+func (t *tracer) initializeHostIP() {
+	t.hostIP = net.ParseIP(t.hostIPString)
```

Comment:
Why do we use `net.ParseIP`? 
It is not likely to return the loopback interface?